src/framework/statusDropdown/StatusDropdown.jsx

```diff
@@ -105,14 +121,17 @@ export default class StatusDropdown extends Component {
       name = this.optionToMetaDataMap[option].name.inactive;
     }
 
-    return [
+    const iconAvailable = !!this.optionToMetaDataMap[option].iconType;
```

Comment:
This check maps the `tick` to the icon that represents the action, and if there are no icon to it's left, no `tick` to it's right. 

Maybe it's slightly more semantic if we have a `showTick` property in the `optionToMetaDataMap`? What do you think? 

That way it's more explicit what the condition is all about. Right now it's basically a implicit relationship between having a icon to the right => having `tick` to the left.

Reply:
talking with Arijit we decided this is a bit out of scope of this ticket.